moveit_configs_utils/src/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -0,0 +1,201 @@
+            self.__robot_description
+            + "_kinematics": load_yaml(
+                self._package_path / (file_path or "config/kinematics.yaml")
+            )
+        }
+        return self
+
+    def joint_limits(self, file_path: Optional[str] = None):
+        self.__moveit_configs.joint_limits = {
+            self.__robot_description
+            + "_planning": load_yaml(
+                self._package_path / (file_path or "config/joint_limits.yaml")
+            )
+        }
+        return self
+
+    def moveit_cpp(self, file_path: Optional[str] = None):
+        self.__moveit_configs.moveit_cpp = load_yaml(
+            self._package_path / (file_path or "config/moveit_cpp.yaml")
+        )
+        return self
+
+    def trajectory_execution(
+        self,
+        file_path: Optional[str] = None,
+        moveit_manage_controllers: bool = True,
+    ):
+        self.__moveit_configs.trajectory_execution = {
+            "moveit_manage_controllers": moveit_manage_controllers,
+        }
+        self.__moveit_configs.trajectory_execution.update(
+            load_yaml(
+                self._package_path
+                / (file_path or f"config/{self.__robot_name}_controllers.yaml")
+            )
+        )
+        return self
+
+    def planning_scene_monitor(
+        self,
+        publish_planning_scene: bool = True,
+        publish_geometry_updates: bool = True,
+        publish_state_updates: bool = True,
+        publish_transforms_updates: bool = True,
+    ):
+        self.__moveit_configs.planning_scene_monitor = {
+            # TODO: Fix parameter namespace upstream -- see planning_scene_monitor.cpp:262
+            # "planning_scene_monitor": {
+            "publish_planning_scene": publish_planning_scene,
+            "publish_geometry_updates": publish_geometry_updates,
+            "publish_state_updates": publish_state_updates,
+            "publish_transforms_updates": publish_transforms_updates,
+            # }
+        }
+        return self
+
+    def planning_pipelines(
+        self, default_planning_pipeline: str = None, pipelines: List[str] = None
+    ):
+        if pipelines is None:
+            pipelines = ["ompl"]
+            default_planning_pipeline = pipelines[0]
+        if default_planning_pipeline not in pipelines:
+            raise RuntimeError(
+                f"default_planning_pipeline: `{default_planning_pipeline}` doesn't name any of the input pipelines "
+                f"`{','.join(pipelines)}`"
+            )
+        self.__moveit_configs.planning_pipelines = {
+            "planning_pipelines": pipelines,
+            "default_planning_pipeline": default_planning_pipeline,
+        }
+        for pipeline in pipelines:
+            self.__moveit_configs.planning_pipelines[pipeline] = load_yaml(
+                self._package_path / "config" / (pipeline + "_planning.yaml")
+            )
+        return self
+
+    def moveit_configs(self):
```

Comment:
This function name is not really intuitive to me. Maybe use something like `to_config()` along with `to_dict()`?

Reply:
We decided to use `to_moveit_configs()`